src/octopeer-github/main/Database/EventObject/KeystrokeEvent.ts

```diff
@@ -2,13 +2,19 @@
  * Created by Maarten on 26-05-2016.
  */
 
+const KeystrokeType = {
```

Comment:
Wait wat? It is used in the `EventFactory` at least, but for consistency I'll add a TSLint suppressing comment there :)